neonvm/tools/vm-builder/files/vector.yaml

```diff
@@ -15,9 +15,18 @@ sources:
         excludes: ["*/proc/sys/fs/binfmt_misc"]
     type: host_metrics
     scrape_interval_secs: 1 # default is 15, but we scrape every 5s in autoscaler-agent
+  sql_exporter_metrics:
+    type: prometheus_scrape
+    # sql_exporter listen addr, exporting LFC statistics.
+    # This statistics will only be available when using the neon vm image. A custom image
+    # does not have this exporter by default.
+    endpoints: ["http://127.0.0.1:9399/metrics"]
+    # The data might be delayed as we use the default scrape interval. Some SQLs are not
+    # trivial to execute, and therefore we don't want them to run every second.
```

Comment:
How much work would it be to run this at the same scrape interval? (or at least, <5s?)

I ask because it may make the scaling algorithm tricky (specifically: expecting counters to remain the same, not because nothing's happening, but because metrics are delayed). I'm sure it can be worked around, but it'd be good to weigh the options for tech debt like this.

Reply:
I think we just need to split two sqlexporters (one for quick scrapes, and one for slow scrapes).